docs/spec_draft.md

```diff
@@ -642,6 +644,43 @@ operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.rsqrt
+
+`stablehlo.rsqrt(operand) -> result`
+
+### Semantics
+
+Performs element-wise reciprocal square root operation on `operand` tensor and
+produces a `result` tensor, implementing the `rSqrt` operation from the IEEE-754
+specification. Numeric precision is implementation-defined.
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `operand` | tensor of floating-point or complex element types |
+
+### Results
+
+| Name | Type |
+|-|-|
+| `result` | tensor of floating-point or complex element types |
+
+### Constraints
+
+  * Supported shapes: all static shapes.
+  * `result` must have the same type as that of `operand`.
+
+### Examples
```

Comment:
An example with complex number? Here and below.

Reply:
Done